src/disemvowling.js

```diff
@@ -0,0 +1,3 @@
+export default function disemvowling(sentence){
+  return sentence.split('').filter((character)=>!character.match(/[aeiou ]/)).join('')
+}
```

Comment:
Why is this file included in the PR for Where in the JSON? Only commit changes that correspond to the branch name and commit messages.